Survi.Prevention.Models/DataTransfertObjects/InspectionForDashboard.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace Survi.Prevention.Models.DataTransfertObjects
+{
+    public class InspectionForDashboard
+    {
+		public Guid Id { get; set; }
+        public Guid? IdRiskLevel { get; set; }
+        public string Address { get; set; }
+        public Guid? IdWebuserAssignedTo { get; set; }
+        public Guid IdBatch { get; set; }
+        public string BatchDescription { get; set; }
+        public DateTime VisitDate { get; set; }
+        public string VisitStatus { get; set; }
+        public Boolean VisitNote { get; set; }
+        public Boolean Anomaly { get; set; }
+        public DateTime LastInspection { get; set; }
+        public DateTime LastReport { get; set; }
+        public Guid IdBuilding { get; set; }
+        public Guid? IdLaneTransversal { get; set; }
+        public Guid IdCity { get; set; }
+        public string PostalCode { get; set; }
+        public string Contact { get; set; }
+        public string Owner { get; set; }
+        public string Matricule { get; set; }
+        public Guid? IdUtilisationCode { get; set; }
+        public Guid? IdPicture { get; set; }
+        public decimal BuildingValue { get; set; }
+        public int NumberOfAppartment { get; set; }
+        public int NumberOfBuilding { get; set; }
+        public int NumberOfFloor { get; set; }
+        public string UtilisationDescription { get; set; }
```

Comment:
I know this field exists in the Building's model, but i think it should be removed as using idUtilisationCode is the right way of getting the value.

Reply:
Inside building table, we had "id_utilisation_code" and "utilisation_description" but I don't the difference